config.toml

```diff
@@ -114,14 +121,6 @@ githubbranch = "v1.9.7"
 docsbranch = "release-1.9"
 url = "https://v1-9.docs.kubernetes.io"
 
-[[params.versions]]
-fullversion = "v1.8.4"
-version = "v1.8"
-githubbranch = "v1.8.4"
-docsbranch = "release-1.8"
-url = "https://v1-8.docs.kubernetes.io"
```

Comment:
With this change, does the netlify site also need to be removed as well?

Currently, I can see `v1-7` and `v1-8` are both available.

Reply:
good call, but we can deal with the issue separately (and even after release ...)